tests/AppBundle/Twig/FormatExtensionTest.php

```diff
@@ -97,16 +105,6 @@ public function testFormatDuration(): void
         );
     }
 
-    /**
-     * Format a number.
-     */
-    public function testNumberFormat(): void
-    {
-        static::assertEquals('1,234', $this->formatExtension->numberFormat(1234));
-        static::assertEquals('1,234.32', $this->formatExtension->numberFormat(1234.316, 2));
-        static::assertEquals('50', $this->formatExtension->numberFormat(50.0000, 4));
-    }
-
```

Comment:
Same here. These seem more meaningful and maybe worthy of moving to the bundle.

Reply:
Oh oops, yes I totally remembered that earlier today but forgot to push the code. It's at https://github.com/wikimedia/ToolforgeBundle/pull/14 now.